example/src/simple/component/master.js

```diff
@@ -11,7 +11,7 @@
  *
  */
 
-import utils from './../../../../src/scripts/utils/master';
+import { Component as BasicComponent } from './../../../../src/scripts/component/master';
```

Comment:
@gbabula this is only because it's a same-module example.

In a real application, it would be relative to `g5-component` i.e. `import from `g5-component/src/scripts/utils/master';`

I *am* trying to avoid aliasing at all, because it negatively affects source code portability across build engines/platforms.

If I have some ESM compatible build system, I can build from an ESM source code module, but not if things are aliased. I then have to determine if my build system can import your build system's aliases as well.